common/models/acl.js

```diff
@@ -546,27 +561,32 @@ module.exports = function(ACL) {
    * @param {Function} cb Callback function
    */
   ACL.isMappedToRole = function(principalType, principalId, role, cb) {
+    cb = cb || utils.createPromiseCallback();
```

Comment:
as it will be in https://github.com/strongloop/loopback/pull/3146, i let you the floor